core/templates/dev/head/pages/exploration_player/conversation_skin_embed_directive.html

```diff
@@ -50,8 +50,11 @@
         <div ng-if="activeCard.inputResponsePairs[activeCard.inputResponsePairs.length - 1].oppiaResponse &&
                     !isCurrentSupplementalCardNonempty() && isCurrentCardAtEndOfTranscript() && !waitingForOppiaFeedback && activeCard.destStateName &&
                     !isOnTerminalCard()">
-          <md-button aria-label="" class="oppia-learner-confirm-button protractor-test-continue-to-next-card-button" focus-on="<[::CONTINUE_BUTTON_FOCUS_LABEL]>" ng-click="showPendingCard(upcomingStateName, upcomingParams, upcomingContentHtml)" translate="I18N_PLAYER_CONTINUE_BUTTON">
-          </md-button>
+          <continue-button focus-on="<[::CONTINUE_BUTTON_FOCUS_LABEL]>"
+                           ng-click="showPendingCard(upcomingStateName, upcomingParams, upcomingContentHtml)">
+          </continue-button>
+          <!-- <md-button aria-label="" class="oppia-learner-confirm-button protractor-test-continue-to-next-card-button" focus-on="<[::CONTINUE_BUTTON_FOCUS_LABEL]>" ng-click="showPendingCard(upcomingStateName, upcomingParams, upcomingContentHtml)" translate="I18N_PLAYER_CONTINUE_BUTTON">
```

Comment:
Drop this completely? Don't leave commented-out code in your PR.

Reply:
Sorry, this is a mistake!